examples/tutorials/audio_data_augmentation_tutorial.py

```diff
@@ -227,40 +90,79 @@ def get_noise_sample(*, resample=None):
 # Apply effects
 waveform2, sample_rate2 = torchaudio.sox_effects.apply_effects_tensor(waveform1, sample_rate1, effects)
 
-print_stats(waveform1, sample_rate=sample_rate1, src="Original")
-print_stats(waveform2, sample_rate=sample_rate2, src="Effects Applied")
+print(waveform1.shape, sample_rate1)
+print(waveform2.shape, sample_rate2)
 
 ######################################################################
 # Note that the number of frames and number of channels are different from
 # those of the original after the effects are applied. Let’s listen to the
 # audio.
 #
 
+def plot_waveform(waveform, sample_rate, title="Waveform", xlim=None):
+    waveform = waveform.numpy()
+
+    num_channels, num_frames = waveform.shape
+    time_axis = torch.arange(0, num_frames) / sample_rate
+
+    figure, axes = plt.subplots(num_channels, 1)
+    if num_channels == 1:
+        axes = [axes]
+    for c in range(num_channels):
+        axes[c].plot(time_axis, waveform[c], linewidth=1)
+        axes[c].grid(True)
+        if num_channels > 1:
+            axes[c].set_ylabel(f"Channel {c+1}")
+        if xlim:
+            axes[c].set_xlim(xlim)
+    figure.suptitle(title)
+    plt.show(block=False)
+
+######################################################################
+#
+
+def plot_specgram(waveform, sample_rate, title="Spectrogram", xlim=None):
+    waveform = waveform.numpy()
+
+    num_channels, _ = waveform.shape
+
+    figure, axes = plt.subplots(num_channels, 1)
+    if num_channels == 1:
+        axes = [axes]
+    for c in range(num_channels):
+        axes[c].specgram(waveform[c], Fs=sample_rate)
+        if num_channels > 1:
+            axes[c].set_ylabel(f"Channel {c+1}")
+        if xlim:
+            axes[c].set_xlim(xlim)
+    figure.suptitle(title)
+    plt.show(block=False)
+
 ######################################################################
 # Original:
-# ~~~~~~~~~
+# ---------
```

Comment:
I prefer the original section organization with the subsections (left) over this updated version (right). imo the lack of indentation makes it look cluttered and harder to navigate, thoughts?

<img width="391" alt="Screen Shot 2022-05-16 at 10 05 24 AM" src="https://user-images.githubusercontent.com/16568633/168611117-13c86494-0ca9-4869-af27-19f9775a7e48.png">